tests/test_utils.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+
+import pandas as pd
+import logging
+from autofocus.util import discard_duplicate_rows
```

Comment:
Please do a quick flake8 run. I think it will want a line before the "from" import two lines before the class.